### PR TITLE
Remove GAed feature gates OpenAPIV3

### PR DESCRIPTION
**Pull request by tukwila**
What type of PR is this?

/kind cleanup

What this PR does / why we need it:

All of these feature gates have been GAed, and can be removed since v1.29:

- OpenAPIV3

Fixes # None

Special notes for your reviewer:

release note
```release-note
Remove GAed feature gates OpenAPIV3
```
Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```
- [KEP]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/3515-kubectl-explain-openapiv3/README.md
```

### CODE REVIEW

**Comment by swatisehgal** [comment]
Please add a release note and link the associated KEP in the additional documentation section.

File: staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go [hunk 1]
```diff
@@ -141,11 +141,9 @@ func (o *WardleServerOptions) Config() (*apiserver.Config, error) {
 	serverConfig.OpenAPIConfig.Info.Title = "Wardle"
 	serverConfig.OpenAPIConfig.Info.Version = "0.1"
 
-	if utilfeature.DefaultFeatureGate.Enabled(features.OpenAPIV3) {
```
Comment (swatisehgal) [inline staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go:144]: We are no longer using `features` so we need to remove the corresponding import `k8s.io/apiserver/pkg/features` in this file above.

Comment (tukwila) [reply staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go:144]: Corrected.

**Comment by tukwila** [comment]
@jpbetz Could you look at this PR?

**Review by swatisehgal — COMMENTED** [review]
/lgtm

**Pull request merged**